changelogs/fragments/1641-case-sensitivity-zos_operator.yml

```diff
@@ -0,0 +1,4 @@
+minor_changes:
+  - zos_operator - add new option case_sensitive to module, allowing users
```

Comment:
nit-picking here, in the release notes (which are built from the changelog), we tend to capitalize first word, use past tense, and put some formatting around any module options mentioned
> Added a new option ``case_sensitive`` to ...

Reply:
Will make those changes